client/components/tinymce/plugins/wpcom-view/views/embed/view.jsx

```diff
@@ -49,12 +33,12 @@ class EmbedView extends Component {
 	}
 
 	constrainEmbedDimensions() {
-		if ( ! this.refs.iframe ) {
+		if ( ! this.iframe ) {
```

Comment:
`this.iframe` is always an object, for example, `{ current: null }`. What you really want to check for is `! this.iframe.current`.